### PR TITLE
Draft delete

### PR DESCRIPTION
**Pull request by AbijithS-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
1851
# Changes

Added draft delete feature and also added a discard button inside edit draft.
# How has the change been tested?

# Screenshots (if applicable)

image
image
image

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by alanraju-aot — APPROVED** [review]
LGTM

File: forms-flow-web/src/components/Draft/DraftOperations.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Link } from "react-router-dom";
+import { MULTITENANCY_ENABLED } from "../../constants/constants";
+import { setDraftDelete } from "../../actions/draftActions";
+
+const DraftOperations = ({ row }) => {
+  const dispatch = useDispatch();
+  const tenantKey = useSelector((state) => state.tenants?.tenantId);
+  const redirectUrl = MULTITENANCY_ENABLED ? `/tenant/${tenantKey}/` : "/";
+  const url = `${redirectUrl}form/${row.formId}/draft/${row.id}/edit`;
+  const buttonText = <Translation>{(t) => t("Edit")}</Translation>;
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/DraftOperations.js:13]: Translation and t is both required

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Draft/Edit.js [hunk 2]
```diff
@@ -193,6 +249,13 @@ const View = React.memo((props) => {
         className="col-12"
       >
         <div className="ml-4 mr-4">
+          <Confirm
+            modalOpen={draftDelete.modalOpen}
+            message={`Are you sure you wish to delete the draft "${draftDelete.draftName}"
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/Edit.js:254]: Translate the message in i18n

File: forms-flow-web/src/components/Draft/List.js [hunk 3]
```diff
@@ -163,6 +193,13 @@ export const DraftList = React.memo(() => {
     >
       {(props) => (
         <div className="container" role="definition">
+          <Confirm
+          modalOpen={draftDelete.modalOpen}
+          message={`Are you sure you wish to delete the draft "${draftDelete.draftName}"
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/List.js:198]: Translation changes

File: forms-flow-web/src/components/Draft/table.js [hunk 4]
```diff
@@ -34,24 +35,33 @@ const linkDraftDetail = (cell, row, redirectUrl) => {
...
-            <i className={icon} />
-            &nbsp;
-          </span>
-          {buttonText}
-        </span>
-      </div>
-    </Link>
-  );
-};
+// const linkDraft = (cell, row, redirectUrl) => {
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/table.js:38]: Can remove the code if unsused

**Review by abhilash-aot — CHANGES_REQUESTED** [review]
Looks good.  Check on translation changes.

File: forms-flow-web/src/components/Draft/DraftOperations.js (hunk 1 above)
Comment (abhilash-aot) [reply forms-flow-web/src/components/Draft/DraftOperations.js:13]: Also test cases to be added

**Pull request closed without merging**